.github/workflows/publish-python.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  pypi-publish:
+    name: Publish release to PyPI
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write  # IMPORTANT: mandatory for sigstore
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
```

Comment:
While it's likely less relevant in this case, we might want to use the earliest Python version that the package officially supports to bulid the distribution. In `setup.cfg`, we have:
```
python_requires = >=3.9
```

Reply:
this has been changed to 3.9, I picked 3.11 because thats what was in the docker container originally.

https://github.com/hunyadi/md2conf/blob/master/Dockerfile#L1